examples/qa_with_sources/client/client.py

```diff
@@ -31,25 +31,17 @@ def main():
         print(colored("Awaiting results. Please be patient. This may take a few moments.", "blue"))
 
         response = api.invoke("/qa_with_sources", query=query)
-        print(colored("Answer: ", "blue"), f"{response['output_text'].strip()}")
+        print(colored("Answer: ", "blue"), f"{response['result'].strip()}")
```

Comment:
just so i follow along properly: this is the result of switching to `VectorDBQAWithSourcesChain` from `load_qa_with_sources_chain` ?

Reply:
Yes! But, I still have to test this out before I can claim this works.

Reply:
i assume this is now verified.